projects/samples/robotbenchmark/square_path/protos/SquarePathPioneer3dx.proto

```diff
@@ -4,7 +4,7 @@
 # The Pioneer 3-DX robot (Adept MobileRobots) is an all-purpose base, used for research and applications involving mapping, teleoperation, localization, monitoring, reconnaissance and other behaviors.
 
 PROTO SquarePathPioneer3dx [
-  field SFVec3f    translation     0 0.0975 0                       # Is `Transform.translation`.
+  field SFVec3f    translation     0 0 0.0975                      # Is `Transform.translation`.
```

Comment:
Changes to the PROTO header should also be updated in the corresponding documentation of the robot